pkgs/by-name/gh/gh-notify/package.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, fetchFromGitHub
+, stdenvNoCC
+, makeWrapper
+, fzf
+, python3
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "gh-notify";
+  version = "0-unstable-2024-03-19";
+
+  src = fetchFromGitHub {
+    owner = "meiji163";
+    repo = "gh-notify";
+    rev = "0d8fa377d79cfef0f66d2f03a5921a5e598e6807";
+    hash = "sha256-Ao6gUtgW7enVlWBQhlQDc8ZW/gP90atc2F4rDNUnjj8=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  installPhase = ''
+    install -D -m755 "gh-notify" "$out/bin/gh-notify"
+  '';
+
+  postInstall = ''
+    wrapProgram "$out/bin/gh-notify" --prefix PATH : "${lib.makeBinPath [ fzf python3 ]}"
```

```suggestion
    wrapProgram "$out/bin/gh-notify" --prefix PATH : "${lib.makeBinPath [ fzf python3 gh ]}"
```

Do you think gh should be added here?
Or
Should it be also considered a standalone command, or only available as a plugin for gh?

Also `delta` and `bat` serves as optional highlight pager in https://github.com/meiji163/gh-notify/blob/0d8fa377d79cfef0f66d2f03a5921a5e598e6807/gh-notify#L268. We're better defaulting to one of them, and add an option providing the other or not providing it at all.

---

All the already other packaged `gh` extensions are Go module which import the core `gh` module so they are working as standalone commands.

I think it would indeed be nice to package `gh-notify` as a standalone command for consistency. We should then also do the same for the recently packaged `gh-f`.

---

In this way we should also wrap gh since the script needs that. Also check other commands in the script.

---

Just added `gh` and `grep` in the wrapper.